.storybook/mock/followers.tsx

```diff
@@ -1,6 +1,9 @@
-import { TwitchUsersFollowsResponse } from '../../interfaces';
+import { useTwitchFollowers } from '../../lib/hooks';
 
-const followers: Record<string, TwitchUsersFollowsResponse> = {
+const followers: Record<
+  string,
+  Required<ReturnType<typeof useTwitchFollowers>>['data']
```

Comment:
Let's export and use `TwitchUsersFollowsResponse` here, no? Let's do the same in other files in this PR.